randovania/games/prime1/gui/preset_settings/prime_patches_qol.py

```diff
@@ -39,6 +39,30 @@ def __init__(self, editor: PresetEditor, game_description: GameDescription, wind
         self.cutscene_combo.setItemData(1, LayoutCutsceneMode.COMPETITIVE)
         self.cutscene_combo.setItemData(2, LayoutCutsceneMode.MINOR)
         self.cutscene_combo.setItemData(3, LayoutCutsceneMode.MAJOR)
+        self.cutscene_combo.setItemData(4, LayoutCutsceneMode.SKIPPABLE)
+
+        if editor._options.experimental_settings:
+            # ruff made me do it this way
+            self.cutscene_label.setText("""
+                <html>
+                    <head/>
+                    <body>
+<p><span style=" font-weight:600;">Original</span>
+: No changes to the cutscenes are made.</p>
+<p><span style=" font-weight:600;">Competitive</span>
+: Similar to minor, but leaves a few rooms alone where skipping cutscenes would be inappropriate for races.</p>
+<p><span style=" font-weight:600;">Minor</span>
+: Removes cutscenes that don't affect the game too much when removed.</p>
+<p><span style=" font-weight:600;">Major</span>
+: Allows you to continue playing the game while cutscenes happen.</p>
+<p><span style=" font-weight:700;">Skippable</span>
+: Keeps all of the cutscenes in the game, but makes it so that they can be skipped with the START button.</p>
+                    </body>
+                </html>
+            """)
+        else:
+            self.cutscene_combo.removeItem(4)
```

Comment:
shouldn't this else block also reset the label to its original text?

Reply:
A current limitation of experimental mode is that it's changes do not apply until the preset editor tab is changed.